test/e2e/flask/multichain-api/get-session.spec.ts

```diff
@@ -0,0 +1,80 @@
+import _ from 'lodash';
+import { withFixtures } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import { DEFAULT_FIXTURE_ACCOUNT } from '../../constants';
+import {
+  DEFAULT_OPTIONS,
+  getExpectedSessionScope,
+  getSessionScopes,
+  openDappAndConnectWallet,
+} from './testHelpers';
+
+describe('Multichain API', function () {
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` when there is no existing session', function () {
+    it('should successfully receive empty session scopes', async function () {
+      await withFixtures(
+        {
+          title: this.test?.fullTitle(),
+          fixtures: new FixtureBuilder().withPopularNetworks().build(),
+          ...DEFAULT_OPTIONS,
+        },
+        async ({
+          driver,
+          extensionId,
+        }: {
+          driver: Driver;
+          extensionId: string;
+        }) => {
+          await openDappAndConnectWallet(driver, extensionId);
+          const parsedResult = await getSessionScopes(driver);
+
+          if (!_.isEqual(parsedResult.sessionScopes, {})) {
+            throw new Error('Should receive empty session scopes');
+          }
+        },
+      );
+    });
+  });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` when there is an existing session', function () {
+    it('should successfully receive result that specifies its permitted session scopes for selected chains', async function () {
+      await withFixtures(
+        {
+          title: this.test?.fullTitle(),
+          fixtures: new FixtureBuilder()
+            .withPopularNetworks()
+            .withPermissionControllerConnectedToTestDapp()
```

Comment:
🤔 I'm confused how this works in this context. I would assume we would need to create a new fixture helper method with CAIP-25 typed permissions added to PermissionsController state?

Reply:
The fixture helper already has CAIP-25 typed permissions. test-dapp and test-dapp-multichain seem to both be served on port 8080 locally, so this permission would apply to both. `wallet_getSession` also doesn't care about the `isMultichainOrigin` flag and will happily return sessionScopes. So yeah, that's why this is working as-is right now. I think that's fine, right?

Reply:
oh lol I was looking at a local branch where this helper still had a legacy permission format